NDTensors/src/Unwrap/src/Unwrap.jl

```diff
@@ -0,0 +1,16 @@
+module Unwrap
+using SimpleTraits
+using LinearAlgebra
+using Base: ReshapedArray
+using Strided: StridedView
```

Comment:
Can use the more minimal [StridedViews.jl](https://github.com/Jutho/StridedViews.jl) which just defines `StridedView` that is used by `Strided.jl`.

Reply:
I think I fixed this using `Strided.StridedViews` is that what you mean?